apps/nextjs-website/react-components/types/Video/Video.types.ts

```diff
@@ -0,0 +1,47 @@
+import { CommonProps, Generic } from '../common/Common.types';
+
+export interface VideoImageProps extends CommonProps {
+  /** Video url */
+  src?:
+    | string
+    | {
+        alternativeText: string | null;
+        url: string;
+        mime: string;
+      };
+  /** Video starts automatically as soon is visible */
+  autoplay?: boolean;
+  loop?: boolean;
+  title?: string;
+  subtitle?: string;
+  caption?: string;
+  /** When true the full layout is used. Default: false */
+  full?: boolean;
+  /** when the full layout is not used, this props change the layout alignment.
+   * Default: false (left aligned layout)
+   */
+  reverse?: boolean;
+  /** String or a custome element to be shown when something goes wrong */
+  fallback?: string | Generic;
+  /** Play button text */
+  playButtonLabel?: string;
+  isCentered: boolean;
+  showVideo?: boolean;
+  imagealt?: string;
+  imagesrc?: string;
```

Comment:
Couldn't we use the "src" prop and add a check on the mime type to determine whether it's an img or a video?

That way we could make `src` required and remove `imagealt` and `imagesrc`.

(If we do, we should keep `alternativeText` in it and disregard the above suggestion of course.)

Reply:
Also, `imagealt` and `imagesrc` are currently never passed, so they are completely useless atm